From ec318fd7a1fa5395abe5cbd45d96f817212c4ed5 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 7 Dec 2012 17:14:36 +0800
Subject: [PATCH 375/449] ENGR00236196: mxc_v4l2_output: add YV12 format
 support

Add the YV12 support in output test to keep consistent with capture test,
Otherwise YV12 recorded video cannot be displayed.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_output.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index 398284d..224e257 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -463,6 +463,7 @@ int process_cmdline(int argc, char **argv)
 			     (g_in_fmt != V4L2_PIX_FMT_YUV422P) &&
 			     (g_in_fmt != IPU_PIX_FMT_YUV444P) &&
                              (g_in_fmt != V4L2_PIX_FMT_YUV420) &&
+                             (g_in_fmt != V4L2_PIX_FMT_YVU420) &&
                              (g_in_fmt != IPU_PIX_FMT_TILED_NV12) &&
                              (g_in_fmt != IPU_PIX_FMT_TILED_NV12F) &&
                              (g_in_fmt != V4L2_PIX_FMT_NV12) )
-- 
1.8.0

